pages/wards/cancel-visit-success.js

```diff
@@ -0,0 +1,127 @@
+  });
+
+  if (hasError || hasDeleteError) {
+    return <Error />;
+  }
+
+  return (
+    <Layout title="Virtual visit has been cancelled">
+      <GridRow>
+        <GridColumn width="full">
+          <form onSubmit={onSubmit}>
+            <Heading>Virtual visit has been cancelled</Heading>
+            <p>The following virtual visit has been successfully cancelled.</p>
+            <dl className="nhsuk-summary-list">
+              <div className="nhsuk-summary-list__row">
+                <dt className="nhsuk-summary-list__key">Patient's name</dt>
+                <dd className="nhsuk-summary-list__value">{patientName}</dd>
+              </div>
+
+              <div className="nhsuk-summary-list__row">
+                <dt className="nhsuk-summary-list__key">Key contact's name</dt>
+                <dd className="nhsuk-summary-list__value">{contactName}</dd>
+              </div>
+
+              <div className="nhsuk-summary-list__row">
+                <dt className="nhsuk-summary-list__key">
+                  Key contact's mobile number
+                </dt>
+                <dd className="nhsuk-summary-list__value">{contactNumber}</dd>
+              </div>
+
+              <div className="nhsuk-summary-list__row">
+                <dt className="nhsuk-summary-list__key">Date of call</dt>
+                <dd className="nhsuk-summary-list__value">{callDate}</dd>
+              </div>
+
+              <div className="nhsuk-summary-list__row">
+                <dt className="nhsuk-summary-list__key">Time of call</dt>
+                <dd className="nhsuk-summary-list__value">{callTime}</dd>
+              </div>
+            </dl>
+            <Button>Return to virtual visits</Button>
+          </form>
+        </GridColumn>
+      </GridRow>
+    </Layout>
+  );
+};
+
+export const getServerSideProps = propsWithContainer(
+  verifyToken(
+    async ({ query, container, authenticationToken }) => {
+      const { callId } = query;
+      let { scheduledCall, error } = await retrieveVisitByCallId(container)(
+        callId
+      );
+      if (error && !scheduledCall) {
+        scheduledCall = {
+          patientName: "",
+          recipientName: "",
+          recipientNumber: "",
+          callTime: "",
+        };
+      }
+
+      const callTime = formatTime(scheduledCall.callTime);
+      const callDate = formatDate(scheduledCall.callTime);
+
+      let { success, error: deleteError } = await deleteVisitByCallId(
+        container
+      )(callId);
+      return {
+        props: {
+          wardId: authenticationToken.ward,
+          patientName: scheduledCall.patientName,
+          contactName: scheduledCall.recipientName,
+          contactNumber: scheduledCall.recipientNumber,
+          callTime,
+          callDate,
+          callId,
+          error,
+          deleteError,
+        },
+      };
+    },
+    {
+      tokens: new TokenProvider(process.env.JWT_SIGNING_KEY),
```

Comment:
Maybe?

```suggestion
      tokens: container.getTokenProvider(),
```

Reply:
Not entirely sure it has access at this stage - Dan is going to be looking into the auth shortly so will probably make a sensible change here